caffe2/contrib/tensorrt/tensorrt_op_trt.cc

```diff
@@ -15,7 +15,7 @@ namespace {
 // Otherwise, return the product of CHW dimensions
 int64_t CheckDims(
     const nvinfer1::Dims& nv_dims,
-    const std::vector<int64_t>& c2_dims) {
+    const at::ArrayRef<int64_t>& c2_dims) {
```

Comment:
ArrayRef is a pass-by-value reference type, so there's no need to reference it again. Just say `at::ArrayRef<int64_t> c2_dims`

Reply:
Thanks!